src/components/Workout/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as CreateWorkoutTask } from './create-workout-task';
+export { default as EditWorkoutTask } from './edit-workout-task';
```

Comment:
is this another export file

Reply:
yes, removed and consolidated with the original one.